vue-app/src/components/UI/SidebarLink.vue

```diff
@@ -26,17 +33,29 @@ a {
   display: grid;
   grid-template-columns: 1fr 2fr 4fr 1fr;
 }
+
 li {
   list-style-type: none;
   width: 100%;
+  max-height: 5vh;
```

Comment:
This property causes the list items to overlap.
The same issue exists on the react page, also due to the max-height.
![image](https://github.com/user-attachments/assets/ae59910b-77c5-498b-8fb1-15c8bace3a4b)

Reply:
this is not issue for this branch, I will create another for the fix for that